src/__test__/actions/users.action.test.js

```diff
@@ -88,4 +92,20 @@ describe('Asset Action tests', () => {
       });
     });
   });
+
+  it('should dispatch CREATE_SECURITY_USER_SUCCESS when addSecurityUser is called successfully', () => {
+    mock.onPost(url3).reply(201, SecurityUser);
+    return store.dispatch(addSecurityUser()).then(() => {
+      expect(store.getActions()[0].type).toEqual(CREATE_SECURITY_USER_SUCCESS);
+      expect(store.getActions()[1].type).toEqual(UPDATE_TOAST_MESSAGE_CONTENT);
```

Comment:
This should not be part of this test case. Remember, always apply  **_"Single Responsibility"_** principle when writing tests. Therefore, this `expect` should be a different test case. E.g.
```
it('should dispatch UPDATE_TOAST_MESSAGE_CONTENT when a security user is successfully created', async () => {
  mock.onPost(url3).reply(201, SecurityUser);
  await store.dispatch(addSecurityUser());
  
  expect(store.getActions()).toContainEqual({
    type: UPDATE_TOAST_MESSAGE_CONTENT,
    payload: <whatever_the_payload_is>
  });
});